t/plugin/udp-logger.t

```diff
@@ -410,10 +383,9 @@ the mock backend is hit
                                 "port": 8125,
                                 "tls": false,
                                 "log_format": {
+                                    "case name": "logger format in plugin",
                                     "vip": "$remote_addr"
-                                },
-                                "batch_max_size": 1,
```

Comment:
We should not remove the `batch_max_size` field

Reply:
that must have been by mistake. I was testing few things out. I'll remove it.